# Status of GNSS aiding
# custom gps_aiding_status_t
	uint8 channel_1  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_2  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_3  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_4  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_5  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_6  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_7  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_8  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_9  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_10  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_11  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 channel_12  # 0 = Source Disabled | 1 = Invalid | 2 = Rejected | 3 = Successfully Processed
	uint8 TimeBiasRepartitionEvent  # 0 = No Event Detected | 1 = Event Detected
